json-sports-team/index.js

```diff
@@ -1,178 +1,257 @@
-      "Position": "MF",
-      "Number": 12,
-      "isCaptain": false,
-      "Nickname": "El Negro"
+      Name: "Héctor Enrique",
+      Position: "MF",
+      Number: 12,
+      isCaptain: false,
+      Nickname: null,
     },
     {
-      "Name": "Oscar Garré",
-      "Position": "DF",
-      "Number": 13,
-      "isCaptain": false,
-      "Nickname": null
+      Name: "Oscar Garré",
+      Position: "DF",
+      Number: 13,
+      isCaptain: false,
+      Nickname: null,
     },
     {
-      "Name": "Ricardo Giusti",
-      "Position": "MF",
-      "Number": 14,
-      "isCaptain": false,
-      "Nickname": null
+      Name: "Ricardo Giusti",
+      Position: "MF",
+      Number: 14,
+      isCaptain: false,
+      Nickname: null,
     },
     {
-      "Name": "Luis Islas",
-      "Position": "GK",
-      "Number": 15,
-      "isCaptain": false,
-      "Nickname": "El loco"
+      Name: "Luis Islas",
+      Position: "GK",
+      Number: 15,
+      isCaptain: false,
+      Nickname: "El loco",
     },
     {
-      "Name": "Julio Olarticoechea",
-      "Position": "DF",
-      "Number": 16,
-      "isCaptain": false,
-      "Nickname": null
+      Name: "Julio Olarticoechea",
+      Position: "DF",
+      Number: 16,
+      isCaptain: false,
+      Nickname: null,
     },
     {
-      "Name": "Pedro Pasculli",
-      "Position": "FW",
-      "Number": 17,
-      "isCaptain": false,
-      "Nickname": null
+      Name: "Pedro Pasculli",
+      Position: "FW",
+      Number: 17,
+      isCaptain: false,
+      Nickname: null,
     },
     {
-      "Name": "Nery Pumpido",
-      "Position": "GK",
-      "Number": 18,
-      "isCaptain": false,
-      "Nickname": null
+      Name: "Nery Pumpido",
+      Position: "GK",
+      Number: 18,
+      isCaptain: false,
+      Nickname: null,
     },
     {
-      "Name": "Oscar Ruggeri",
-      "Position": "DF",
-      "Number": 19,
-      "isCaptain": false,
-      "Nickname": "El Cabezón"
+      Name: "Oscar Ruggeri",
+      Position: "DF",
+      Number: 19,
+      isCaptain: false,
+      Nickname: "El Cabezón",
     },
     {
-      "Name": "Carlos Tapia",
-      "Position": "MF",
-      "Number": 20,
-      "isCaptain": false,
-      "Nickname": null
+      Name: "Carlos Tapia",
+      Position: "MF",
+      Number: 20,
+      isCaptain: false,
+      Nickname: null,
     },
     {
-      "Name": "Marcelo Trobbiani",
-      "Position": "MF",
-      "Number": 21,
-      "isCaptain": false,
-      "Nickname": "Calesita"
+      Name: "Marcelo Trobbiani",
+      Position: "MF",
+      Number: 21,
+      isCaptain: false,
+      Nickname: "Calesita",
     },
     {
-      "Name": "Héctor Zelada",
-      "Position": "GK",
-      "Number": 22,
-      "isCaptain": false,
-      "Nickname": null
-    }
-  ]
+      Name: "Héctor Zelada",
+      Position: "GK",
+      Number: 22,
+      isCaptain: false,
+      Nickname: null,
+    },
+  ],
+};
+
+// this could be a good review of destructuring
```

Comment:
If we decide to repurpose this project to teach more modern JS techniques, this should be the first time destructuring is introduced.